reactDashboard/src/App.css

```diff
@@ -1,83 +1,164 @@
+header {
+  padding-top: 10px;
 }
 
-.link {
-  color: #3c3c3b;
-  font-weight: bold;
-  text-decoration: none;
+.darkmode header {
+  background-color: #121212;
+}
+
+/* color is default white for .region-selector-label */
+.darkmode .region-selector-label {
+  color: #f2f2f2;
+}
+
+.heatmaps-row,
+.region-page-top-row {
+  padding: 15px 10px;
+  background-color: #f2f2f2;
+}
+
+.darkmode .heatmaps-row {
+  background-color: #262626;
+}
+
+.darkmode .region-page-top-row {
+  background-color: rgb(56, 56, 56);
+}
+
+.geo-map-column {
+  padding-left: 5px;
+  padding-right: 5px;
+  margin-bottom: 5px;
+}
+
+/* color is default white for .region-tiles-column */
+.region-tiles-column {
+  padding-top: 0;
+  margin-bottom: 5px;
+}
+
+.darkmode .region-tiles-column {
+  background-color: #121212;
+}
+
+.underHeatmapSelector {
+  border: 1px solid #2a84bb;
+}
+
+.darkmode .underHeatmapSelector {
+  border: 1px solid #cccccc;
 }
 
+/* color and background-color are default for .about-us, .accessibility-page, .data-sources */
 .about-us,
 .accessibility-page,
 .data-sources {
+  padding: 0 15px;
   text-align: left;
-  padding-top: 3vh;
-  padding-right: 15px;
-  padding-left: 15px;
-}
-
-.data-sources {
-  padding-bottom: 3vh;
 }
 
-.about-us-details,
-.accessibility-intro,
-.accessibility-details {
-  padding-bottom: 30px;
+.darkmode .about-us,
+.darkmode .accessibility-page,
+.darkmode .data-sources {
+  background-color: #121212;
+  color: #f2f2f2;
 }
 
 .about-us h1,
 .about-us h2,
 .accessibility-page h1,
+.accessibility-page h2,
 .data-sources h1 {
-  padding-top: 2vh;
+  font-size: 1rem;
+  padding: 15px;
+  background-color: #133a53;
+  color: white;
+  border-radius: 5px;
+  font-weight: bold;
 }
 
-.accessibility-page ul {
-  list-style-type: none;
-  margin: 0 0 1rem 0;
-  padding: 0;
+.data-sources-list {
+  min-height: 52.5vh;
+  padding: 20px 40px;
 }
 
-.accessibility-intro li {
-  padding-left: 20px;
+/* background-color is default for .accessibility-details li, .about-us-details li, .data-sources-list li, .about-us-details ul, .accessibility-details ul, .data-sources .data-sources-list */
+.accessibility-details li,
+.about-us-details li {
+  padding: 0 20px 0;
 }
 
-.accessibility-page .browsers {
-  font-style: italic;
-  padding-left: 20px;
-  margin-bottom: 15px;
+.darkmode .accessibility-details li,
+.darkmode .about-us-details li,
+.darkmode .data-sources-list li,
+.darkmode .about-us-details ul,
+.darkmode .accessibility-details ul,
+.darkmode .data-sources .data-sources-list {
+  background-color: rgb(50, 50, 50);
 }
 
-.accessibility-page #browsers-list {
-  margin-bottom: 5px;
+.about-us-details ul {
+  width: 50vw;
 }
 
-.about-us .name {
-  color: #3c3c3b;
-  font-size: 16pt;
+.accessibility-details ul {
+  padding-left: 50px;
+  padding-bottom: 20px;
 }
 
-.team-members {
-  justify-content: left;
+/* background-color is default for .accessibility-details p, .about-us-details p */
+.darkmode .accessibility-details p,
```

Comment:
Put this block after the block it overrides (the next one down)

Reply:
✅